local-modules/doc-client/EditorComplex.js

```diff
@@ -32,7 +32,7 @@ const DEFAULT_BODY_MODULE_CONFIG = {
   keyboard: BayouKeyHandlers.defaultKeyHandlers,
   toolbar: [
     [{ header: 1 }, { header: 2 }, { header: 3 }],
-    ['bold', 'italic', 'underline', 'strike', 'code'],
+    ['bold', 'italic', 'underline', 'strike', 'link', 'code'],
```

Comment:
Oops! This ended up attributed to the wrong commit. (The original change was made in a different non-public repo and got shunted over here via semi-automatic scriptery which I did not sufficiently double-check!)